manifests/listen/all.pp

```diff
@@ -68,11 +68,27 @@
   Enum['ipv4','ipv6','all','auto'] $apply_to     = 'auto',
   Simplib::Netlist                 $trusted_nets = simplib::lookup('simp_options::trusted_nets', { 'default_value' => ['127.0.0.1'] })
 ){
-  iptables_rule { "all_${name}":
-    first    => $first,
-    absolute => $absolute,
-    order    => $order,
-    apply_to => $apply_to,
-    content  => template("${module_name}/allow_all_services.erb")
+  include 'iptables'
+
+  if $iptables::use_firewalld {
+    $metadata = {
+      'dports'       => $dports,
```

Comment:
$dports is not defined in this manifest.

Reply:
Good catch, thanks.

Reply:
Re-reading, I just realized that I didn't handle this case properly. It's not used by SIMP by default so was missed.